app/javascript/components/Reporting/index.js

```diff
@@ -105,35 +95,39 @@ const tableExporter = (startDate, endDate, onStartChange, onEndChange, state, ma
   )
 
   const octetStream = encodeURIComponent(csv)
+  const today = new Date()
```

Comment:
Do we want the date being updated every re-render? Or just per page load - then we can call it outside the function / useEffect (for once only).

Reply:
well, its kind of inexpensive and there is the edge case where the component mounts before 12am and reloads after 12am 🤷‍♀